subworkflows/local/sv_germline_benchmark.nf

```diff
@@ -4,17 +4,18 @@
 
 params.options = [:]
 
-include { TRUVARI_PHAB           } from '../../modules/local/truvari_phab'                  addParams( options: params.options )
+include { TRUVARI_PHAB           } from '../../modules/local/truvari_phab'             addParams( options: params.options )
 include { TRUVARI_BENCH          } from '../../modules/nf-core/truvari/bench'          addParams( options: params.options )
 include { SVANALYZER_SVBENCHMARK } from '../../modules/nf-core/svanalyzer/svbenchmark' addParams( options: params.options )
-include { WITTYER                } from '../../modules/nf-core/wittyer'                  addParams( options: params.options )
-include { VCFDIST                } from '../../modules/local/vcfdist'                  addParams( options: params.options )
+include { WITTYER                } from '../../modules/nf-core/wittyer'                addParams( options: params.options )
+include { TABIX_BGZIP as TABIX_BGZIP_QUERY } from '../../modules/nf-core/tabix/bgzip'  addParams( options: params.options )
+include { TABIX_BGZIP as TABIX_BGZIP_TRUTH } from '../../modules/nf-core/tabix/bgzip'  addParams( options: params.options )
```

Comment:
Why not handle this in the modules.config?

Reply:
what do you mean?

Reply:
differentiating between TABIX_BGZIP_QUERY and TABIX_BGZIP_TRUTH? one module cannot be used twice in a workflow. That is the main reason to do this..

Reply:
I see